core/components/styles/sidebar.module.css

```diff
@@ -43,6 +45,10 @@
   display: none;
 }
 
+.bottomButtons {
+  display: none;
```

Comment:
There is tailwind for this  https://tailwindcss.com/docs/display 

Display none, flex, all this "positioning" styles are good to use for flex.

Reply:
Separated task https://www.notion.so/tianlu/Use-SCSS-and-sync-all-the-hardcode-colores-and-documented-c72c7e12a1e9439b9bcc5bac020dcf8a?pvs=4